packages/mui-icons-material/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/icons-material",
-  "version": "5.2.5",
+  "version": "5.2.9",
```

Comment:
Are we supposed to bump this package?

Reply:
See https://github.com/mui-org/material-ui/pull/30667#discussion_r785806389